thirdparty/runtime_environments_test.go

```diff
@@ -0,0 +1,20 @@
+package thirdparty
+
+import (
+	"context"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetImageNames(t *testing.T) {
```

Comment:
This test is going to fail on the container variant (see my [patch here](https://spruce.mongodb.com/version/6682bbbde1096400081a6ab9/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)), presumably because the containers cannot communicate with the API. Winston made it so that the task hosts can talk to the API, but containers are a bit different.

Instead of changing our container configuration (which would end up being a lot of effort), I think we should just skip the `test-thirdparty` task on the container. There's an example of this in `self-tests.yml` using [task tags](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files#task-and-variant-tags). You can reuse the task tag we already have called `skip-container`. Take a look and see if this makes sense, and we can talk about it at office hours in a bit!